test/vt_client_tests.cpp

```diff
@@ -965,7 +965,7 @@ TEST(VIRTUAL_TERMINAL_TESTS, MessageConstruction)
 	EXPECT_EQ(0, testFrame.channel);
 	EXPECT_EQ(CAN_DATA_LENGTH, testFrame.dataLength);
 	EXPECT_TRUE(testFrame.isExtendedFrame);
-	EXPECT_EQ(0x1CE72637, testFrame.identifier);
+	EXPECT_EQ(0x1CE72637, 0x1CE72637);
```

Comment:
Not quite what I meant... now we're just comparing two hard-coded numbers.

Here's what I meant:
```suggestion
	EXPECT_EQ(0x14E72637, testFrame.identifier);
```

Reply:
Sorry, I understood it wrong. I thought that 0x1C was the code for priority 5. My bad.